style.css

```diff
@@ -1,469 +1,490 @@
-  align-items: center;
-  width: 327px;
-  position: absolute;
-  top: 6385px;
-  left: 50%;
-  transform: translate(-50%, -50%);
-}
-
-.social-media2 {
-  list-style: none;
-  display: flex;
-  flex-direction: row;
-  gap: 20px;
-  margin-top: 20px;
-  padding: 0 1rem;
-  align-items: center;
-  justify-content: center;
-}
-
-.stack {
-  display: flex;
-  justify-content: space-between;
-  flex-direction: row;
-  gap: 24px;
-  padding-top: 20px;
-  font-size: 12px;
-  align-items: flex-start;
-}
-
-.items {
-  color: #36b37e;
-  padding: 8px 12px;
-  background-color: white;
-  border-radius: 2px;
-}
-
-.figure1 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  position: relative;
-  top: 3320px;
-  left: 150px;
-  margin-left: auto;
-  margin-right: auto;
-}
-
-.figure2 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  position: absolute;
-  top: 4270px;
-  margin-right: auto;
-  margin-left: auto;
-}
-
-.figure3 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  position: relative;
-  top: 5465px;
-  margin-right: 0.1px;
-  margin-left: auto;
-}
-
-@font-face {
-  font-family: 'Inter-for-Microverse';
-  src: url('inter.ttf') format('truetype');
-}
-
-.form-container ul {
-  list-style: none;
-  padding-left: 0;
-  width: 100%;
-}
-
-.form-container input {
-  font-size: 20px;
-}
-
-textarea {
-  height: 80px;
-  font-size: 20px;
+@media only screen and (min-width: 768px) and (max-width: 2442px) {
+  /* --GENERAL SHAPE -- */
+  body {
+    margin: 0;
+    width: 100%;
+  }
+
+  @font-face {
+    font-family: 'Inter-for-Microverse';
+    src: url('inter.ttf') format('truetype');
+  }
+
+  /* --NAVIGATION MENU-- */
+
+  .direction {
+    margin-top: 15px;
+    display: flex;
+    justify-content: center;
+    gap: 30px;
+    color: #172b4d;
+    padding-top: 0.6rem;
+    padding-bottom: 0.6rem;
+  }
+
+  /* --TOP SECTION-- */
+  .info {
+    display: flex;
+    justify-content: center;
+    margin-top: 100px;
+  }
+
+  .info h1 {
+    font: 800 48px "Inter", sans-serif;
+    letter-spacing: 0.37px;
+    text-align: center;
+    color: #172b4d;
+    margin-top: 70px;
+    margin-bottom: auto;
+  }
+
+  .info h2 {
+    font: 800 48px/60px "Inter", sans-serif;
+    letter-spacing: 0.37px;
+    text-align: center;
+    color: #36b37e;
+    margin: auto;
+  }
+
+  .info p {
+    font-family: "Inter", sans-serif;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  /* --TOP SECTION BACKGROUND-- */
+  .top-figures {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    height: 1407px;
+    width: 1442px;
+    background-image: url(images/topfigures.svg);
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: absolute;
+    z-index: 1;
+    top: 1px;
+  }
+
+  .header {
+    display: flex;
+    justify-content: center;
+  }
+
+  .header a {
+    text-decoration: none;
+  }
```

Comment:
- Great Job for removing the underline, I think it would be better if you update the color also to stick the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=yfbn9fW3IN2AUzsN-0) color `#344563`  and have better UI👌